jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/log/ProcessInstanceLogPresenter.java

```diff
@@ -153,81 +141,43 @@ public boolean onError(ClientRuntimeError error) {
-                agent = constants.Human();
-            }
-
-            return Optional.of(LOG_TEMPLATES.getTechLog(DateUtils.getDateTimeStr(logSummary.getDate()),
-                                                        logSummary.getNodeType(),
-                                                        SafeHtmlUtils.fromString(logSummary.getName()),
-                                                        (logSummary.isCompleted() ? " " + constants.Completed() : ""),
-                                                        agent).asString());
-        } else {
-            String prettyTime = DateUtils.getPrettyTime(logSummary.getDate());
-
-            if (NODE_HUMAN_TASK.equals(logSummary.getNodeType())) {
-                return Optional.of(LOG_TEMPLATES.getBusinessLog(prettyTime,
-                                                                constants.Task(),
-                                                                SafeHtmlUtils.fromString(logSummary.getName()),
-                                                                (logSummary.isCompleted() ? constants.WasCompleted() : constants.WasStarted())).asString());
-            } else if (NODE_START.equals(logSummary.getNodeType()) && !logSummary.isCompleted()) {
-                return Optional.of(LOG_TEMPLATES.getBusinessLog(
-                        prettyTime,
-                        constants.Process(),
-                        SafeHtmlUtils.fromString(getProcessName()),
-                        constants.WasStarted()).asString());
-            } else if (NODE_END.equals(logSummary.getNodeType()) && logSummary.isCompleted()) {
-                return Optional.of(LOG_TEMPLATES.getBusinessLog(
-                        prettyTime,
-                        constants.Process(),
-                        SafeHtmlUtils.fromString(getProcessName()),
-                        constants.WasCompleted()).asString());
-            }
-        }
-        return Optional.empty();
+    public void loadMoreProcessInstanceLogs() {
+        setCurrentPage(currentPage + 1);
+        loadProcessInstanceLogs();
     }
 
-    public void onProcessInstanceSelectionEvent(@Observes final ProcessInstanceSelectionEvent event) {
-        setProcessName(event.getProcessDefName());
-        setServerTemplateId(event.getServerTemplateId());
+    public void resetLogsList() {
+        currentPage = 0;
+        visibleLogs = new ArrayList<>();
+    }
 
-        view.setActiveLogOrderButton(LogOrder.ASC);
-        view.setActiveLogTypeButton(LogType.BUSINESS);
+    public void loadTaskDetails(Long workItemId,
+                                ProcessInstanceLogHumanTaskView humanTaskView) {
+        taskService.call(
+                (TaskSummary task) -> {
+                    humanTaskView.setDetailsData(task);
+                }).getTask(serverTemplateId,
+                           containerId,
+                           workItemId);
```

Comment:
@nmirasch I think you probably need to call another service here, one that finds the task based on the _workItemId_,  check if _UserTaskServicesClient.findTaskByWorkItemId_ can help with that.

Reply:
@cristianonicolai about showing extra TaskInstance data like, potential owners, business admins, excluded owners and createdby. All this info is only filled by UserTaskServicesClient.getTaskInstance. We can use this call instead UserTaskServicesClient.findTaskByWorkItemId (it don't fill that info) you suggested if we want to show that information and add an extra field TaskAssignmentSummary to TaskSummary to make available that info. Make it sense?